eksupgrade/src/preflight_module.py

```diff
@@ -124,7 +124,8 @@ def get_cluster_version(
                 return
         cmk_key_check(errors, cluster_name, region, cluster_details, report, customer_report)
         security_group_check(errors, cluster_name, region, cluster_details, report, customer_report)
-        pod_security_policies(errors, cluster_name, region, report, customer_report)
+        if float(update_version) < 1.25:
```

Comment:
```suggestion
        if float(cluster_details["cluster"]["version"]) < 1.25:
```
to target the current cluster version LT 1.25